### PR TITLE
Fix intro typo

### PR DESCRIPTION
**Pull request by thepetk**
## What does this PR do?

This PR simply fixes a typo in the `data/home/intro.md`

## Which issue(s) does this PR fix

N/A

### CODE REVIEW

File: data/home/intro.md [hunk 1]
```diff
@@ -7,4 +7,4 @@ volunteer-led Python programming conference on the planet! Join us in July in
 the vibrant Bohemian city Prague. We'll be together, celebrating our shared passion for Python and its community!
 
 Call for Proposals, Mentorship Programm, Financial Aid Program
```
Comment (hypha) [inline data/home/intro.md:9]:
```suggestion
Call for Proposals, Mentorship Programme, Financial Aid Programme
```

Comment (thepetk) [reply data/home/intro.md:9]: That thing :D

**Review by hypha — APPROVED** [review]
high fives, fellow typo hunter!

**Comment by artcz** [comment]
Thank you!

**Pull request merged**